### ISSUE TITLE
Fresh (offline) installation fails

### ISSUE THREAD

**Issue by B0ycee** [openhab/openhab-distro#961]
Attempting to start openhab on a Linux based device without internet access having extracted runtime distribution .zip and placed addons .kar in the addons directory:

Using 2.5.0M1 - succeeds
Using 2.5.0M2 (and also latest snapshot) - fails with following:

org.apache.karaf.features.internal.util.MultiException: Error:
        Error downloading mvn:org.openhab.ui.bundles/org.openhab.ui.dashboard/2.5.0-SNAPSHOT
        at org.apache.karaf.features.internal.download.impl.MavenDownloadManager$MavenDownloader.<init>(MavenDownloadManager.java:91)
...

Perhaps this might be a regression associated with not having internet access available?  Someone else has hit the same error ( https://community.openhab.org/t/openhab-not-starting-error-downloading-mvn-org-openhab-ui-bundles-org-openhab-ui-dashboard-2-5-0-m2/ ) although perhaps the resolution by re-installation is due to using an earlier version or having internet access on re-install?

**Comment by kaikreuzer** [comment]
I think, I have found the culprit: It is this change: https://github.com/openhab/openhab-distro/pull/883

This causes Karaf to install the Dashboard even before the KAR is extracted and its content made available.

If you remove that line from `userdata/etc/org.apache.karaf.features.cfg`, the error goes away. As a result, you won't see the Dashboard in a fresh installation, though.

You can either install it on the openHAB console with
```
feature:install openhab-ui-dashboard
```
or, if you are anyhow using textual configuration, add `dashboard` to the `ui=` section in `conf/services/addons.cfg`:

```
# A comma-separated list of UIs to install (e.g. "ui = basic,paper")
ui = dashboard,paper,basic
```

Unfortunately, a fix cannot be included in the 2.5.1 addons, since the issue is within the distro, for which we do not plan to do a 2.5.1 release.

But I hope the above information is a suitable workaround for anybody wanting to do an offline installation.

Issue will be fixed for openHAB 3.

**Comment by bwosborne2** [comment]
> Unfortunately, a fix cannot be included in the 2.5.1 addons, since the issue is within the distro, for which we do not plan to do a 2.5.1 release.

I thought a 2.5.1 patch release was planned.

**Comment by openhab-5iver** [comment]
2.5.1 (and newer 2.5.x) will unfortunately not include updates to OHC and OHD. They have moved on in development towards OH 3.0 and will stay locked at 2.5.0.

**Comment by bwosborne2** [comment]
> 2.5.1 (and n



> 2.5.1 (and newer 2.5.x) will unfortunately not include updates to OHC and OHD. They have moved on in development towards OH 3.0 and will stay locked at 2.5.0.

Even when 2.5 is totally broken for offline use?? That may be stable but I assumed OH was to be stable AND functional. A dumb rock is about as stable and functional as a new 2,5 offline install with addons.

**Comment by bwosborne2** [comment]
Shouldn't the documentation have been updated with the workaround before this Issue was closed?

**Comment by kaikreuzer** [comment]
> Unfortunately, a fix cannot be included in the 2.5.1 addons, since the issue is within the distro, for which we do not plan to do a 2.5.1 release.

After all, I have come up with https://github.com/openhab/openhab-distro/pull/1041 - if the other maintainers agree, we will ship a 2.5.1 distro as a fix for this issue.

**Comment by kaikreuzer** [comment]
This has been fixed with #1041 in openHAB 2.5.1.
